tests/functional/pointerevent_pointerleave_descendants-manual.js

```diff
@@ -0,0 +1,20 @@
+define(function(require) {
+	var registerSuite = require('intern!object');
+	var w3cTest = require('../support/w3cTest');
+	var name = 'pointerevent_pointerleave_descendants-manual';
+
+	registerSuite({
+		name: name,
+
+		main: function() {
+			return w3cTest(this.remote, name + '.html')
+				.findById('target0')
+					.moveMouseTo(50, 8)
+					.end()
+				.findByTagName('body')
+					.moveMouseTo(50, 30)
```

Comment:
Sorry, I am A) still a hot mess and B) get too many notifications from GitHub.

The coordinates in `moveMouseTo` are relative to the top-left corner of the context element (the thing you got using `find*`). So as @scottgonzalez notes this test starts the mouse at 50,8 from the corner of `#target0` and then moves it to 50,30 of `body`, which moves the mouse upwards and never contacts the purple box.

Looking at the test instructions, I suspect a key thing to understand given the direction to “move it out through purple rectangle boundaries” is that you will never get intermediate `mousemove/over/out` events like you would from a hardware mouse device. WebDriver will move the mouse from being in one place in the first tick to being exactly in the requested next place the next tick. If there are elements in between these two points, they won’t be triggered. So while you might intuitively think “oh I can just start the mouse at the top of the document and move it to the bottom of the document and it will move through all the intersecting elements”, in reality it will just always be in the body. You’ll need to add those intermediate positions yourself explicitly.

Hopefully that helps, let me know if I misunderstand something about the question.